src/services/github_service.ts

```diff
@@ -0,0 +1,72 @@
+import {SearchGitHubUsersRequest, GetUsersInSpecificLocationRequest,GitHubUserRepoLanguageRequest } from '../interfaces/github.client';
+import {GitHubGetUsersResponse,GetGitHubLanguagesResponse,GetUsersInSpecificLocationsResponse} from '../interfaces/github.response';
+
+//>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+export const searchGitHubUsers = async (request: SearchGitHubUsersRequest): Promise<GitHubGetUsersResponse> => {
```

Comment:
Please do not move github.client functions here, instead you should call the functions from here.